connectors/yarpc/yarpc.go

```diff
@@ -323,7 +339,11 @@ func (c *Connector) MultiRead(ctx context.Context, ei *dosa.EntityInfo, keys []m
 
 	response, err := c.Client.MultiRead(ctx, request, VersionHeader())
 	if err != nil {
-		return nil, errors.Wrap(err, "MultiRead failed")
+		if !dosarpc.Dosa_MultiRead_Helper.IsException(err) {
+			return nil, errors.Wrap(err, "failed to multiread due to network issue")
```

Comment:
Here "multiread" is not an english word; so it should either be MultiRead (which I believe is what the endpoint is called) or "multi-read". My preference is that it should be the endpoint name, so "MultiRead".

Similarly for the other errors, it would be better to put in the method name exactly, preserving case...

Reply:
updated